.buildkite/scripts/pipelines/security_solution_quality_gate/pipeline.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
-
 set -euo pipefail
 
-echo "In the entrypoint for the quality gate"
\ No newline at end of file
+source .buildkite/scripts/common/util.sh
+source .buildkite/scripts/steps/functional/common_cypress.sh
+.buildkite/scripts/bootstrap.sh
+
+export JOB=kibana-security-solution-chrome
+
+buildkite-agent meta-data set "${BUILDKITE_JOB_ID}_is_test_execution_step" "true"
+
+echo "--- Serverless Security Second Quality Gate"
+cd x-pack/test/security_solution_cypress
+set +e
+
+CYPRESS_ELASTICSEARCH_URL=$TEST_ENV_ES_URL CYPRESS_BASE_URL=$TEST_ENV_KB_URL CYPRESS_ELASTICSEARCH_USERNAME=$TEST_ENV_USERNAME CYPRESS_ELASTICSEARCH_PASSWORD=$TEST_ENV_PWD CYPRESS_KIBANA_URL=$CYPRESS_BASE_URL yarn cypress:run:qa:serverless; status=$?; yarn junit:merge || :; exit $status
+
+.buildkite/scripts/lifecycle/post_command.sh
```

Comment:
Can we add a note for why this is called manually?

Reply:
@dkirchan can you please help with the above comment?

Reply:
@jbudz We have a very first draft integration of our test-suite with Buildkite. Looking at all the rest of the pipelines and the way they are configured, I believe that we will be getting down this way later, to automate the way it is called for consistency. But at the moment, as far as it comes to 1 environment (QA) and one quality gate for our needs, it gives us flexibility to debug and find problems. 


Having said all this, I need to also mention that we are simplifying literally everything for our specific testsuite and needs for the serverless testing for cypress tests. This means that we will be evaluating in each step we will be going through what we already have for other pipelines and of course reuse whatever suits our needs, but also craft new implementation if this is decided by the whole team.

Reply:
Sorry - I meant specifically why `.buildkite/scripts/lifecycle/post_command.sh` is called.  buildkite on kibana-ci will automatically call the lifecycle hooks. 

If this is not using kibana-ci it will be skipped, but it looks out of place and a comment would help.

Reply:
@jbudz Oh I see. I was thinking that these hooks live only in the `.buildkite/hooks` folder so I originally thought it was a script crafted by us to it for us when we invoke it. 

Thank you for your comment!!

Reply:
The line has been removed, @jbudz may you please take another look when you have the chance? Thanks! :)